Scripts/MNISTdataset.swift

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
I suggest putting the usage comments on the function declarations instead of at the top level of the file (https://google.github.io/swift/#file-comments).

Also, I have a few more comment formatting requests from https://google.github.io/swift/#documentation-comments:
* Use "///" for documentation comments instead of "/* ... */"
* Use Apple's markup format (https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/). In particular, surround code examples with three backticks.

(same comment applies to a few other comments that you have before declarations in this file)